lib/gems/pending/appliance_console/database_configuration.rb

```diff
@@ -272,5 +274,9 @@ def initialize_from_hash(hash)
         end
       end
     end
+
+    def default_port
```

Comment:
You can always override the port which is an `attr_accessor`, but I don't see any need to override the default. Kinda would defeat the purpose of a default ...